pkg/query/api/v1.go

```diff
@@ -185,6 +188,21 @@ func (api *API) parseEnableDedupParam(r *http.Request) (enableDeduplication bool
 	return enableDeduplication, nil
 }
 
+func (api *API) parseReplicaLabelsParam(r *http.Request) (replicaLabels []string, _ *ApiError) {
+	const replicaLabelsParam = "replicLabeals[]"
```

Comment:
typo

So it means we don't have test for this in `v1_test.go` (: Can we add test for this?